torch/_subclasses/functional_tensor.py

```diff
@@ -140,6 +140,7 @@ def __new__(cls, elem):
             False,  # dispatch_device
             False,  # dispatch_layout
             extra_dispatch_keys,  # _extra_dispatch_keys
+            elem.untyped_storage().nbytes(),  # storage_size
```

Comment:
Supplying ```storage_size``` explicitly to avoid the invocation of ```computeStorageNbytes``` which has checks on the individual sizes of ```elem.shape``` being passed in.

Reply:
This is OK, send this on its own, test would be nice but not strictly needed.

Reply:
Done